__tests__/components/Comments/WriteComment.test.js

```diff
@@ -104,7 +104,7 @@ describe('WriteComment should run correctly', () => {
       backgroundColor: COLORS.balance_gray,
       position: 'absolute',
       top: 0,
-      left: 30,
+      left: 37,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The left position has been increased from 30 to 37. This could potentially shift the element out of view or overlap with other elements on smaller screens. Please ensure that this change doesn't cause any UI issues on different screen sizes.
```diff
- left: 30,
+ left: 37,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->